data/remote/src/main/java/com/ku_stacks/ku_ring/remote/user/UserClient.kt

```diff
@@ -20,4 +20,12 @@ class UserClient @Inject constructor(
 
     suspend fun registerUser(token: String): DefaultResponse =
         userService.registerUser(RegisterUserRequest(token))
+
+    suspend fun getKuringBotQueryCount(token: String): Int {
+        return try {
+            userService.getKuringBotQueryCount(token).data.leftAskCount
+        } catch (e: Exception) {
+            0
+        }
+    }
```

Comment:
```suggestion
    suspend fun getKuringBotQueryCount(token: String): Int {
        return runCatching {
            userService.getKuringBotQueryCount(token).data.leftAskCount
        }.getOrElse { 0 }
    }
```

이렇게 수정해도 됩니다! 개인적으로 이게 좀 더 가독성 좋게 보이긴한데 취향선인 것 같아요

Reply:
저도 이게 더 깔끔해 보이네요. 앞으로 try-catch 대신 `runCatching`을 주로 써볼까 합니다

https://github.com/ku-ring/KU-Ring-Android/pull/339/commits/907f1e59d276dcee3870109c3ed2d50eaf2b12a8 에서 수정했습니다.